src/app/user/join/[teamID]/page.js

```diff
@@ -18,10 +18,10 @@ export default function page({ params }) {
       body: { team: params.teamID },
     }).then((response) => {
       if (response.message !== "OK") {
-        toast(`❌ ${response.message}`);
+        toaster(`❌ ${response.message}`, "error");
         return;
       }
-      toast("✅ Successfully joined team!");
+      toaster("✅ Successfully joined team!", "success");
```

Comment:
icon should be implied by the type of toast

Reply:
Do I remove the icons?